libraries/rush-lib/src/index.ts

```diff
@@ -191,3 +191,7 @@ export {
   type IRushCommandLineParameter,
   type IRushCommandLineAction
 } from './api/RushCommandLine';
+
+export { RushGlobalFolder } from './api/RushGlobalFolder';
+
+export { Autoinstaller, type IAutoinstallerOptions } from './logic/Autoinstaller';
```

Comment:
I'm not sure it's valuable to expose these through API. Installation operations are finicky.